bin/gather_logs.sh

```diff
@@ -47,11 +47,9 @@ sort_by_last() {
 	-o 'UserKnownHostsFile /dev/null')
     logs=(/var/log /etc)
     logs+=(/var/chef/cache /var/cache/chef /opt/dell/crowbar_framework/db)
-    curlargs=(-o /dev/null -D - --connect-timeout 30 --max-time 120 --insecure \
-    --location)
-    [[ $CROWBAR_KEY ]] && curlargs+=(--digest -u "$CROWBAR_KEY")
-    for to_get in nodes proposals roles; do
-	curl "${curlargs[@]}" "http://localhost/$to_get" || :
+    crowbarctl node list
+    for to_get in proposals roles; do
+        crowbarctl $to_get proposal list crowbar
```

Comment:
This is of course not 1:1 change, but I do not really understand the original meaning of those calls ...is it to just generate some activity in the log files?